### PR TITLE
mw commit

### PR DESCRIPTION
**Pull request by modnarwolf**

### CODE REVIEW

File: modnar-wolf-wtwb.html [hunk 1]
```diff
@@ -0,0 +1,348 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="shortcut icon" type="image/x-icon" href="./favicon.ico">
+    <title>CryptoGateway Interview w/ ModnarWolf: Where The Wind Blows</title>
+
+    <meta name="description" content="Interview w/ Steve Pikelny: Fake Internet Money">
```
Comment (scyclow) [inline modnar-wolf-wtwb.html]: you need to change all the metadata so it shows up properly on social media

File: modnar-wolf-wtwb.html [hunk 2]
```diff
@@ -326,7 +326,8 @@ <h2 style="text-align: center">Where The Wind Blows will be available for purcha
         
         <div class="question">
           <p><strong>CG: That's actually pretty fascinating, I'll likely go down the nautical flag rabit whole. 
-            It has been a great chat ModnarWolf. I look forward to seeing the mints and how it all evolves. 
+            It has been a great chat ModnarWolf. Hope to ha you back here some day and I look forward to
```
Comment (scyclow) [inline modnar-wolf-wtwb.html]: typo: `ha` -> `have`

File: modnar-wolf-wtwb.html [hunk 3]
```diff
@@ -7,23 +7,23 @@
     <link rel="shortcut icon" type="image/x-icon" href="./favicon.ico">
     <title>CryptoGateway Interview w/ ModnarWolf: Where The Wind Blows</title>
 
-    <meta name="description" content="Interview w/ Steve Pikelny: Fake Internet Money">
-    <meta name="twitter:image" content="./img/FIM-testnet-30000264.png">
-    <meta name="og:image" property="og:image" content="./img/FIM-testnet-30000264.png">
+    <meta name="description" content="Interview w/ ModnarWolf: Where The Wind Blows">
+    <meta name="twitter:image" content="./img/mw-img/wtwb-img-01.jpg">
+    <meta name="og:image" property="og:image" content="./img/mw-img/wtwb-img-01.jpg">
     <meta name="og:image:alt" content="Fake Internet Money #0">
```
Comment (scyclow) [inline modnar-wolf-wtwb.html]: missed one

File: modnar-wolf-wtwb.html (hunk 2 above)
Comment (modnarwolf) [reply modnar-wolf-wtwb.html]: hoooppss (^ _  ^)

**Pull request merged**